requirements.txt

```diff
@@ -0,0 +1,19 @@
+adal==1.2.7
```

Comment:
top-level requirements shouldn't be checked in since it's generated by the generate_requirements.py script.